### DISCUSSION TITLE
Is there a way to adjust the delivery attempts?

### DISCUSSION THREAD

**Discussion by k040711** [postalserver/postal#2429] [Help with using Postal] [answered]
Hi,

I went over the docs but could not find the answer, hence posting here...

Is there a way to adjust a postal setting ( or from bash ) to adjust postal to not attempt to deliver more than once ( or twice ) ?

Right now its set to 18, but I want to reduce it.

Thanks :)

**Comment by sinnbeck** [comment, marked as answer]
Yeah it's very easy. Open your postal.yml file and add/change this line

https://github.com/postalserver/postal/blob/main/config/postal.defaults.yml#L14

Then restart postal
```
postal stop && postal start

**Reply by k040711** [reply]
thank you !  , that was super easy :)

**Reply by uhlhosting** [reply]
> Yeah it's very easy. Open your postal.yml file and add/change this line
> 
> 
> 
> https://github.com/postalserver/postal/blob/main/config/postal.defaults.yml#L14
> 
> 
> 
> Then restart postal
> 
> ```
> 
> postal stop && postal start 

Wow. Its just mindblowing how many variables in postal config that were not included in default configs i use.
